app/src/main/java/org/schabi/newpipe/util/DrawableResolver.kt

```diff
@@ -0,0 +1,26 @@
+package org.schabi.newpipe.util
+
+import android.content.Context
+import android.graphics.drawable.Drawable
+import androidx.annotation.AttrRes
+
+/**
+ * Utility class for resolving [Drawables](Drawable)
+ */
+class DrawableResolver {
+    companion object {
+        @JvmStatic
+        fun resolveDrawable(context: Context, @AttrRes attrResId: Int): Drawable? {
+            return androidx.core.content.ContextCompat.getDrawable(
+                context,
+                android.util.TypedValue().apply {
```

Comment:
I think it would be better if these were replaced with imports instead of manually having them be written out.